hikari/impl/event_manager.py

```diff
@@ -370,6 +375,7 @@ async def on_guild_create(  # noqa: C901, CFQ001 - Function too complex and too
             self._auto_chunk_members
             and self._intents & intents_.Intents.GUILD_MEMBERS
             and (payload.get("large") or not presences_declared)
+            and (self._cache is None or not self._cache.settings.only_my_member)
```

Comment:
This disables auto_chunk_members when using the `only_my_member` option.
Maybe `auto_chunk_members` should be changed to None by default, which means "auto", so that it can still be explicitly enabled by passing `True` (defaults to `True` currently)